src/misc_utils.jl

```diff
@@ -118,15 +125,7 @@ Calculate element-wise residuals
 ```
 """
 @inline function calculate_residuals(u₀, u₁, α, ρ, internalnorm)
-  @. calculate_residuals(u₀, u₁, α, ρ, internalnorm)
-end
-
-@inline function calculate_residuals(u₀::Array{T}, u₁::Array{T}, α::T2,
-                                             ρ::Real, internalnorm) where
-                                             {T<:Number,T2<:Number}
-    out = similar(u₀)
-    calculate_residuals!(out, u₀, u₁, α, ρ, internalnorm)
-    out
+  @loop u₀ (u₁ - u₀) / (α + max(internalnorm(u₀), internalnorm(u₁)) * ρ)
```

Comment:
It seems the test failures indicate that this returns `nothing`?

Reply:
And similarly https://github.com/JuliaDiffEq/OrdinaryDiffEq.jl/pull/639/files#diff-35dc3e015c5ed3a7fd9f2ef3be5afd55R116